datumaro/plugins/yolo_format/extractor.py

```diff
@@ -156,14 +162,23 @@ def name_from_path(cls, path: str) -> str:
 
         return osp.splitext(path)[0]
 
+    @classmethod
+    def _image_loader(cls, *args, **kwargs):
+        return load_image(*args, **kwargs, keep_exif=True)
```

Comment:
I like that this change is local for yolo, but at the same time other formats are probably suffering from this as well.

Reply:
Yes, I would suggest merging this hotfix first, and then implement a more universal solution